teku/src/test/java/tech/pegasys/teku/cli/BeaconNodeCommandTest.java

```diff
@@ -450,7 +450,7 @@ private Path createConfigFile() throws IOException {
 
     return expectedConfigurationBuilder()
         .eth2NetworkConfig(b -> b.applyNetworkDefaults("mainnet"))
-        .executionEngine(b -> b.endpoints(new ArrayList<String>()))
+        .executionEngine(b -> b.endpoint(null))
```

Comment:
should this be an Optional? its shaping up like that...

Reply:
Internally it is but I don't think it makes sense for the param to be an Optional as the one place in production code it's actually called is the CLI options and PicoCLI doesn't really do Optional so it's going to get a null from there so makes sense for the builder to take care of the conversion since builders are meant to make building easier.

This test is just a bit weird in that it starts from something with a whole bunch of values changed then resets them back to defaults. Hence we're generally migrating away from `BeaconNodeCommandTest`...